setup.cfg

```diff
@@ -1,21 +0,0 @@
-[metadata]
-version = attr: simplemonitor.__version__
-
-[flake8]
-max-line-length = 88
-ignore = E501,W503,E203,W503
-per-file-ignores =
-    simplemonitor/Monitors/__init__.py:F401
-    simplemonitor/Loggers/__init__.py:F401
-    simplemonitor/Alerters/__init__.py:F401
-    simplemonitor/__init__.py:F401
-
-[isort]
-known_third_party = arrow,boto3,botocore,freezegun,importlib_metadata,jinja2,markupsafe,oauthlib,paho,paramiko,psutil,pyaarlo,requests,ring_doorbell,sphinx,sphinx_rtd_theme,twilio,win32event,win32service,win32serviceutil
-known_first_party = Alerters,Loggers,Monitors,envconfig,util,simplemonitor
-line_length=88
-multi_line_output=3
-include_trailing_comma=True
```

Comment:
not all of these have config equivalents in ruff, but doesn't seem to cause any complaints in terms of existing formatting.